src/test/java/com/yourcompany/SampleSauceTest.java

```diff
@@ -17,7 +19,7 @@
 import org.openqa.selenium.OutputType;
 
 
-@TestObject(testObjectApiKey = "INSERT_API_KEY_HERE", testObjectSuiteId = INSERT_SUITE_NUMBER_HERE)
+@TestObject(testObjectApiKey = "INSERT_API_KEY_HERE", testObjectSuiteId = 12345)
```

Comment:
@ndmanvar the testObjectSuiteId is automatically overridden in this case (we are listening for the environment variable within our library). Putting a random value there at least allows us to override it without running into an error when we try to compile. You can only insert constant values in annotations, too, so we are kind of limited in how we approach this.